tool/clean/clean_util.go

```diff
@@ -7,6 +7,7 @@ import "time"
 
 const (
 	KeepDurationOneWeek        = KeepDurationOneDay * 7
+	KeepDurationFourDays       = KeepDurationOneDay * 4
```

Comment:
Just curious but is there a reason we pick 4 days over 7 days or we just pick this number for more frequent clenaups?

Reply:
Yeah the latter. Just to make it more frequent.